Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -135,6 +140,15 @@ + (void)didReceiveLocalNotification:(UILocalNotification *)notification
                                                     userInfo:details];
 }
 
+
+
++ (void)didFailToRegisterForRemoteNotificationsWithError:(NSError *)error
```

Comment:
Can you the update the documentation in https://github.com/facebook/react-native/blob/c6020a0ef4880bead15a0ed2f4058a70cc9b6ad5/Libraries/PushNotificationIOS/PushNotificationIOS.js#L73 to mention this method.